configs/recommended-script.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @fileoverview the `recommended-script` config for `eslint.config.js`
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+"use strict"
+
+const { configs, rules } = require("../lib/index.js")
+
+module.exports = {
+    plugins: { n: { rules } },
+    languageOptions: { sourceType: "commonjs" },
```

Comment:
Note that for `sourceType: "commonjs"`, ESLint  automatically enables only [`exports`, `global`, `module`, and `require`](https://github.com/eslint/eslint/blob/559ff4e4bc54a8b6e6b54825d83c532d724204b3/conf/globals.js#L12-L17) globals, so this config probably needs to enable more globals here to match what its eslintrc version was doing.

Reply:
addressed in 0f22186cd206ce139df6e1b03f2e364d9bb644b6